packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -70,6 +70,18 @@ export const MovableLine = (props: Props) => {
                     end={endPtPx}
                     style={{stroke: "transparent", strokeWidth: TARGET_SIZE}}
                 />
+                <SVGLine
+                    start={startPtPx}
+                    end={endPtPx}
+                    className="movable-line-focus-outline"
+                    style={{}}
+                />
+                <SVGLine
+                    start={startPtPx}
+                    end={endPtPx}
+                    className="movable-line-focus-outline-gap"
+                    style={{}}
```

Comment:
Given that `style` is made optional in this PR, is there a reason we need/want to explicitly pass an empty `style` here?